### DISCUSSION TITLE
Click to call

### DISCUSSION THREAD

**Discussion by AlexTryHarder** [osTicket/osTicket#6195] [Ideas]
I wonder if there is a way to append "tel://" to phone number in ticket, so instead of editing field browser would make a phone call to ticket owner
![image](https://user-images.githubusercontent.com/14946862/177965700-937010b8-9a18-404a-9a8f-5f0f88c4ec47.png)

**Comment by ntozier** [comment]
You could use this PR to add the phone back to the ticket view, and it would be pretty easy to prefix the html "tel://".

https://github.com/osTicket/osTicket/pull/4392

Also a some newer cloud based VOIP phone systems have a dial key where you can highlight the number and press a key and it will dial.

**Reply by AlexTryHarder** [reply]
Thanks, I managed to create href pointing to a User phone number, however, need to access the phone number submitted during ticket creation,
I assumed it would be $ticket->Num()  [Varible assigned in form] but it keeps breaking my ticket view.
Any idea how I could refer to that variable?

**Reply by AlexTryHarder** [reply]
So I end up with code like that:
`						foreach (DynamicFormEntry::forTicket($ticket->getId()) as $form) {}
						echo $form->getTitle();
						$link_address = $ticket->getPhoneNumber(); 
                        echo "<div class='contact-number'><a onclick=\"ga('send', 'event','phone number', 'click to call', '$link_address', 1);\" href='tel:$link_address'>Call on Rainbow ( $link_address )</a></div>";
                        ?>`
                        
Very messy, but it;'s allowing me to get the title of the custom form, now I just need to call that one variable

![image](https://user-images.githubusercontent.com/14946862/178024888-4ad210a6-6866-4339-9547-de7a6b315456.png)

**Reply by AlexTryHarder** [reply]
@ntozier Can you please point me in the right direction?

**Reply by ntozier** [reply]
If your using a dynamic form instead of the built in phone / extension I dont know how you would do that.

**Comment by Mike-Benoit** [comment]
Also see #5318 for very similar functionality. Unfortunately the osTicket team doesn't respond well (or in this case at all) to outside PR's though, so its unlikely to ever be accepted.
